api/v1beta1/keystoneapi_types.go

```diff
@@ -231,6 +231,9 @@ type KeystoneAPIStatus struct {
 
 	// NetworkAttachments status of the deployment pods
 	NetworkAttachments map[string][]string `json:"networkAttachments,omitempty"`
+
+	// Memcached instance name.
+	MemcachedInstance string `json:"memcachedInstance,omitempty"`
```

Comment:
Let's understand why we would need this  and what're the use-cases?  I don't expect this to be changed in production deployments.

Reply:
This is the field in status, not the one in spec. Because we have to find out the instance previously used, so I decided to record it in the spec.

Talking about the spec.MemcachedInstance field, that follows the common approach to handle shared services such as db or mq. I agree we 'd avoid having different approach specific to memcached.

Reply:
Ack, I did not know that we've already added that in spec. I don't think we should all these assuming someone would flip between memached instances.

Reply:
If we don't assume people would not flip between memcached instance, then we should prohibit that operation (by adding validation at webhook). I can add the validation, but that would also trigger the discussion about the potential migration scenario (similarly to db, though we less likely to see the use case specific to memcached).

Because the fix is light, I'd prefer keeping the capability now.